### PR TITLE
add changelog and a warning for is_signature_valid on crl

### PR DESCRIPTION
**Pull request by reaperhulk**

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -32,6 +32,8 @@ Changelog
   :attr:`~cryptography.x509.RFC822Name.value` attribute was deprecated, users
   should use :attr:`~cryptography.x509.RFC822Name.bytes_value` to access the
   raw value.
+* Added :attr:`~cryptography.x509.CertificateRevocationList.is_signature_valid`
```
Comment (alex) [inline CHANGELOG.rst]: Wait, this shouldn't be attr, it should be meth.

Comment (reaperhulk) [inline CHANGELOG.rst]: Ugh, we made it a property on CSR and a method on CRL then. Should we just change it to a property on CRL for consistency?

Comment (alex) [inline CHANGELOG.rst]: How could you make it a property, don't you need to pass the issuers public key as an argument?

Comment (alex) [inline CHANGELOG.rst]: CSR works because it's always self-signed.

Comment (reaperhulk) [inline CHANGELOG.rst]: Indeed you do. Maybe I shouldn't suggest things right after waking up.

**Pull request merged**